packages/dht/src/exports.ts

```diff
@@ -7,7 +7,7 @@ export { PeerDescriptor, Message, NodeType, DataEntry } from './proto/packages/d
 export { ITransport } from './transport/ITransport'
 export { ConnectionManager, ConnectionLocker, PortRange, TlsCertificate } from './connection/ConnectionManager'
 export { PeerID, PeerIDKey } from './helpers/PeerID'
-export { DhtPeer } from './dht/DhtPeer'
+export { RemoteDhtNode as DhtPeer } from './dht/RemoteDhtNode'
```

Comment:
Could export the class as `RemoteDhtNode`.

Maybe we don't export the class in the future if our plan is that this class in an internal class? E.g. in Layer1 events we wanted to use PeerDescriptor class instead of this class so that our interface is more abstract. The only place DhtPeer is still used is `ILayer#getNeighborList`.